common/http/src/test/java/io/helidon/common/http/SetCookieTest.java

```diff
@@ -0,0 +1,57 @@
+package io.helidon.common.http;
+
+import org.junit.Test;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.text.SimpleDateFormat;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.Date;
+
+import static org.junit.Assert.*;
+
+public class SetCookieTest {
+
+    private SetCookie setCookie = new SetCookie("foo", "bar");
```

Comment:
Also, you should declare the object under test each time within each test.

Reply:
done